### PR TITLE
initial recipe

### PR DESCRIPTION
**Pull request by cdeepali**
Initial recipe for `av` package. It is required to support video backend in `torchvision`.

### CODE REVIEW

**Review by dncliss — APPROVED** [review]
LGTM; I don't see any issues (noted that a README.md stub was already pushed in separately).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
```
Comment (bnemanich) [inline recipe/meta.yaml]: This should be 1 instead of `{{ build_number }}`

Comment (bnemanich) [reply recipe/meta.yaml]: Oh, I missed that `build_number was defined up above. It would match the rest of our recipes if we don't use a variable here.

Comment (cdeepali) [reply recipe/meta.yaml]: sure, updated it.

**Review by bnemanich — APPROVED** [review]

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**